### PR TITLE
iOS SDK 0.12.2

### PR DESCRIPTION
**Pull request by dgoldman-pdx**
Adds Call Log, Streemshot Editing, Invitations.

@seanadkinson do you want to add some documentation to `README.md` re the new invitation stuff?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,18 +87,23 @@ Inside your `AppDelegate.application(_, didFinishLaunchingWithOptions:)` impleme
...
     public func func measurementUnitDidChange(measurementUnit: UnitLength) {
         // Your app might wish to copy this value to your app preferences,
         // so that when starting future sessions you can restore it
         // (by setting `Streem.sharedInstance.measurementUnit`).
     }
+    
+    public func helpRequested() {
+        // This function relates to an upcoming Streem feature.
+        // For now, you can simply provide a do-nothing stub here.
+    }
```
Comment (dgoldman-pdx) [inline README.md]: This is what I get for taking so long to release this updated SDK...

File: Podfile [hunk 2]
```diff
@@ -4,17 +4,24 @@ source 'https://github.com/streem/cocoapods'
...
+	installer.pods_project.targets.each do |target|
+		target.build_configurations.each do |config|
+			# This allows us to build for use in Swift 5.1.2
+			#
+			# At the moment, we apparently need to include this setting
+			# here in our StreemNow Podfile, to be compatible with the
+			# build settings used by our SDK itself.
+			# Without this setting, symbol-mangling apparently becomes
+			# incompatible between the SDK and this project.
+			config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'YES'
```
Comment (dgoldman-pdx) [inline Podfile]: Yet another side-effect of Apple's new `BUILD_LIBRARY_FOR_DISTRIBUTION` setting. 😿

File: README.md (hunk 1 above)
Comment (seanadkinson) [reply README.md]: Can we not make it an optional delegate method?

File: StreemNow/StreemInitializer.swift [hunk 3]
```diff
@@ -64,4 +80,8 @@ extension StreemInitializer: StreemDelegate {
             defaults.set(string, forKey: defaultsMeasurementUnitsKey)
         }
     }
+
+    public func helpRequested() {
+        print("Help requested.")
+    }
```
Comment (seanadkinson) [inline StreemNow/StreemInitializer.swift]: Remove this if we can make the delegate method optional.

**Comment by dgoldman-pdx** [comment]
Delegate methods now optional. ✅

**Pull request merged**